### DISCUSSION TITLE
Unable to get rid of Dependabot

### DISCUSSION THREAD

**Discussion by WarningImHack3r** [community/community#73072] [Code Security] [answered]
### Select Topic Area

Bug

### Body

I deleted my Dependabot config file from my repo (from a PR that I merged).
To my surprise, Dependabot was still active.
I tried disabling any Dependabot/code security I could find, both in my repo and in my whole organization, but Dependabot is still creating pull requests, no matter what I do.

How can I properly and for good disable Dependabot for a selected repository? Why does it even continue working without a config file?

Thanks for your answer.

EDIT:
- In my repo, `Settings` ➜ `Code security and analysis`, **everything** is disabled. Dependabot still works.
- In my org settings, `Code security and analysis`, in the `Dependabot` section I clicked `Disable all` on both items multiple times, and no rule is enabled.
- In any way, this is **not** a normal behavior for Dependabot to be active **without any configuration file**. No matter what settings are enabled, this should not happen. I shouldn't have to toggle settings that could affect _all_ my repositories only because of this issue in one repository.

**Comment by ktauchathuranga** [comment]
take a look at this, this might help you,
https://github.com/dependabot/feedback/issues/216
happy coding 🖤

**Reply by WarningImHack3r** [reply]
Thanks for your answer but:
- This thread is very outdated, I already read it and nothing is really relevant anymore
- I've already clicked all `Disable all` buttons everywhere in Code Security for both the repo and the organization profile, no luck

**Reply by ktauchathuranga** [reply]
thank you for the update mate 🖤

**Comment by PriyanshNegi** [comment]
@WarningImHack3r This is because you've enabled Depandabot setting for whole organization. As an admin you can disable it from organization page settings. Now for a specific repo, you can disable depandabot :

If you've allowed Dependabot at the organization level but want to disable it for a specific repository, you can follow these exact steps on GitHub:

1. Go to the repository in which you want to disable Dependabot.

2. Click on the "Settings" tab, located on the right side of the repository's menu.

3. In the repository settings, scroll down and click on "Code Security & analysis" in the left menu.

4. You should see a section related to Dependabot. It might be called "Dependabot alerts" or something similar.

5. In this section, you should find a setting to disable Dependabot alerts or code analysis for this repository. There may be an option like "Automatically detect security vulnerabilities" or "Enable Dependabot."

6. Disable this setting to turn off Dependabot for this specific repository.

7. Save your changes.

**Reply by WarningImHack3r** [reply]
Thanks for your answer, however, I've already tried it and it still doesn't change anything. Moreover, this is not a normal behavior for Dependabot to still be active **without any configuration**.

**Comment by darksh71** [comment]
@WarningImHack3r To properly disable Dependabot for a repository:

Check for any lingering Dependabot configuration files and remove them.
Disable Dependabot alerts, code security, and code scanning in the repository settings.
Review and disable any GitHub Actions workflows that might trigger Dependabot updates.
Ensure there are no organization-level settings that override repository settings.
Remove any webhooks related to Dependabot.
If issues persist, contact GitHub support for further assistance.
Dependabot may continue working without a config file due to default settings or previous configurations. Removing configurations and adjusting settings as mentioned above should disable it for good.

**Reply by WarningImHack3r** [reply]
Hi, thanks for your answer. I updated my post.

**Reply by darksh71** [reply]
I totally get your frustration. Dependabot should follow the rules, no matter what. It's a bit like a stubborn koala, but don't worry. GitHub support is your best mate in this situation. They'll help you sort out this issue without affecting all your other repos. Cheers!

**Comment by carogalvin** [comment]
Hi @WarningImHack3r , that's very strange behaviour...could you send me the name of your repo? To clarify, is Dependabot actively opening new pull requests even when `dependabot.yml` is deleted, or is it just not closing its old PRs? I wouldn't expect it to close old ones.

**Reply by WarningImHack3r** [reply]
Hi Caro, thanks for your reply!
The repo is [EmeraldHQ/Website](https://github.com/EmeraldHQ/Website).

It's not only old PRs, it creates new ones even without configuration file.

As an proof, [12 PRs were created](https://github.com/EmeraldHQ/Website/pulls?q=is%3Apr+is%3Aclosed+author%3Aapp%2Fdependabot) although the migration was done [more than 2 weeks ago](https://github.com/EmeraldHQ/Website/pull/153).

Since the replies I received yesterday I tried clicking a few more times the `Disable` buttons here and there, I'm now waiting for some NPM updates to be released to see if the bot opens yet another PR but I'm almost sure it will.

**Reply by carogalvin** [reply, marked as answer]
OK, I can see that Dependabot didn't pick up the deleted file for some reason and was still operating on the previous schedule that you set. I went and re-sync'ed that for you, you should no longer see Dependabot opening PRs for you (unless you re-enable it again someday on purpose). Let me know if that's not the case.

**Reply by WarningImHack3r** [reply]
Thank you very much!! I'll hit you up if I ever get a new PR from Dependabot :)
